### ISSUE TITLE
PS4 Controller unable to connect after pairing

### ISSUE THREAD

**Issue by mealejan** [ValveSoftware/SteamOS#703]
#### Your system information

* Steam client version: Dec 16 2019,
* SteamOS version: 1576550254
* Opted into Steam client beta?: [Yes/No] 
* Opted into SteamOS beta?: [Yes/No] 
* Have you checked for updates in Settings > System?: [Yes/No]

#### Please describe your issue in as much detail as possible:
Describe what you _expected_ should happen and what _did_ happen.

#### Steps for reproducing this issue:

1. Pair PS4 controller
2. Reboot
3. Push PS4 controller button to pair
4. No pairing possible
5. Repeat for Step 3 to infinity

**Comment by kisak-valve** [comment]
Possibly related to https://github.com/ValveSoftware/steam-for-linux/issues/6756.

**Comment by alkazar** [comment]
Yes, this is the same issue as https://github.com/ValveSoftware/steam-for-linux/issues/6756

In the case of the PS4 controller, if you connect the controller via USB and pair it, the controller will be paired correctly and connect automatically even after a reboot.
